libs/exec/execv.go

```diff
@@ -0,0 +1,26 @@
+package exec
+
+import "io"
+
+type ExecvOptions struct {
+	// Args is the name of the command to run and its arguments.
+	// Eg: ["echo", "hello"] for "echo hello"
+	Args []string
+
+	// Env is set the environment variables to set in the child process.
+	Env []string
+
+	// Dir is the working directory of the child process.
+	Dir string
+
+	// Stdin, Stderr and Stdout are only used for Windows where execv is not supported.
+	// For Unix, these are automatically inherited during the exec
```

Comment:
We are now inheriting on Windows as well, can these be cleaned up?

Reply:
Thanks, missed that.